deps/uv/src/win/fs-event.c

```diff
@@ -225,11 +225,10 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   }
 
   if (!handle->buffer) {
-    handle->buffer = (char*)_aligned_malloc(uv_directory_watcher_buffer_size,
-                                            sizeof(DWORD));
+    handle->buffer = (char*)uv__malloc(uv_directory_watcher_buffer_size);
```

Comment:
That was reviewed back in the day and Bert pointed out that plain malloc
provided the necessary alignment in that case, that's why we dropped it.
On Jun 11, 2015 21:05, "Julien Gilli" <notifications@github.com> wrote:

> In deps/uv/src/win/fs-event.c
> <https://github.com/joyent/node/pull/25475#discussion_r32255959>:
>
> > @@ -225,11 +225,10 @@ int uv_fs_event_start(uv_fs_event_t* handle,
> >    }
> >
> >    if (!handle->buffer) {
> > -    handle->buffer = (char*)_aligned_malloc(uv_directory_watcher_buffer_size,
> > -                                            sizeof(DWORD));
> > +    handle->buffer = (char*)uv__malloc(uv_directory_watcher_buffer_size);
>
> @saghul <https://github.com/saghul> IIUC, here the call to _aligned_malloc
> would be replaced by a call to malloc, is it intentional and what is the
> impact of such a change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joyent/node/pull/25475/files#r32255959>.
>